BipedRobot/ZMPBipedRobot/docs/2. Optimisation process/objectives_values_Fs50.txt

```diff
@@ -0,0 +1,3 @@
9102268,0.09749764614343359,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,0.07872890844841011,0.08012920017579533,0.08171382969268541,0.08269996683906308,0.08425059213639126,0.08591160303671908,0.08681908761972942,0.08741838560304148,0.07869378659516539,0.07983846561912833,0.08186229932261659,0.08270585241615742,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,0.07269247868746685,0.07418152591371903,0.07529864370678238,0.07619434978020008,0.07249579602832804,0.07368332065823273,0.07532256004469423,0.0759261260433436,0.06791771890193495,0.06906933475792083,0.07038901635322632,0.07119180402329772,0.14473329003149246,0.1455692091720771,-Inf,-Inf,0.14396559323372515,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,0.06001384960424279,0.06104005543456998,0.06233653342598998,0.06321687842931718,0.059858316756501814,0.06103294634081273,0.06227699369115817,0.06314134338120155,0.06012536181455148,0.061097082189469305,0.06177787573025507,0.06306237190982131,0.05995676685926289,0.060609397893804005,0.0619116865260742,0.06246664366896763,0.11835163099921259,0.11923553120822186,0.12067780571305074,0.121103941034183,0.11863487217349594,0.12018512354749479,0.12030168667761053,0.12088910531073394,0.11916598120287308,0.12029781678312745,0.12052058454798581,-Inf,0.11936284066839915,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,0.05708731918541937,0.05829836728119693,0.05977816067642629,0.06063348383301766,0.05659410548535089,0.05826231323676547,0.059493495387574175,0.06006818783960851,0.05674796259145285,0.057937808143604434,0.058937100780044535,0.05979041844371466,0.05349557165058044,0.054697462201594424,0.05547270618139821,0.05603908477152258,0.11234033874746918,0.11352456437231183,0.11431775078217994,0.11516122278722181,0.11197392203858801,0.11323765858415831,0.11422162554890646,0.11507999388810097,0.1114492079796431,0.1129643646780809,0.11346865925622879,0.1143290641768589,0.10622031523522428,0.10710173359964063,0.10718296224679612,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,0.051137836476190336,0.052454506132691585,0.05370203357556836,0.0549329286455747,0.04777657619875799,0.049366053445992195,0.05028313122602464,0.051185229243640475,0.05053068057275504,0.05183973697202446,0.053445890312542606,0.05388082493758667,0.048112011425124696,0.04942097612954436,0.050133389679184806,0.050812858604264204,0.10108127598597859,0.10253627191993515,0.10327772395424982,0.10356028318105388,0.09504732039432323,0.096186606903401,0.09709494584863348,0.09736818621225184,0.10021590080672503,0.10112866532580465,0.10212499250362808,0.10284368875680962,0.09581646828855402,0.0968115134058904,0.09726785081632323,0.09803796501113807,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf,-Inf
```

Comment:
Same here

Reply:
This file is more important to run the example 3. It can't run the Example 3 (Optimised controller) without it. But we can regenerate it using Example 2 (Optimisation process) but I can be quite long (>1h)